api/krusty/inlinelabels_test.go

```diff
@@ -14,7 +14,13 @@ kind: Deployment
 metadata:
   name: my-deployment
 spec:
+  selector:
+    matchLabels:
+      my: deployment
   template:
+    metadata:
+      labels:
+        my: deployment
```

Comment:
why add these?

Reply:
Explanation is here https://github.com/kubernetes-sigs/kustomize/pull/4209#discussion_r749687663, but basically the test yaml was missing some required fields

Reply:
I still don't understand why you need to modify this string, because the only place that it is used is in TestKustomizationLabels, which we shouldn't be modifying in this PR. 

Why does your PR require changes to an existing test?

Reply:
I pulled your branch and all the tests all pass without this additional change, so I prefer not to change it.

Reply:
ok let me remove it, the test change was before adding the `includeTemplates` flag

Reply:
done 🙂

Reply:
awesome, thanks!